downstairs/src/extent_inner_raw.rs

```diff
@@ -1268,17 +1305,14 @@ impl RawInner {
         for (i, block) in (counter.min_block..=counter.max_block).enumerate() {
             if self.active_context[block] == copy_from {
                 dest_slots[i] = source_slots[i];
-
-                // Mark this slot as unsynched, so that we don't overwrite
-                // it later on without a sync
-                self.context_slot_dirty.set(block, !copy_from);
```

Comment:
Why was this removed? Is it because the next write will call sync after setting dirty and therefore we don't need to keep track of this anymore?